frontend/packages/ux-editor/src/components/leftMenu/ConfigureLayoutSetPanel.tsx

```diff
@@ -0,0 +1,133 @@
+  const configPanelRef = useRef<HTMLDivElement>(null);
+
+  function handleConfigureLayoutSet() {
+    if (layoutSetName === '') {
+      setErrorMessage(t('left_menu.pages_error_empty'));
+    }
+    else {
+      configureLayoutSetMutation.mutate({ layoutSetName });
+    }
+  }
+
+  const handlePopoverOpen = (event: MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+    setPopoverOpen(true);
+  };
+
+  const handlePopoverClose = () => {
+    setAnchorEl(null);
+    setPopoverOpen(false);
+  };
+
+  const handleKeyPress = async (event: any) => {
+    if (event.key === 'Enter') {
+      handleConfigureLayoutSet()
+      setEditLayoutSetName(false);
+    } else if (event.key === 'Escape') {
+      setEditLayoutSetName(false);
+      setLayoutSetName('');
+    }
+  };
+
+  const handleClickOutside = (event: any) => {
+    if (configPanelRef.current && !configPanelRef.current.contains(event.target as Node)) {
+      setEditLayoutSetName(false);
+      setLayoutSetName('');
+    }
+  };
+
+  useEffect(() => {
+    document.addEventListener('mousedown', handleClickOutside);
+    return () => {
+      document.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, []);
+
+  const handleConfigureLayoutSetButtonClick = () => {
+    setEditLayoutSetName(!editLayoutSetName);
+  }
+
+  const handleOnChange = (event: ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+    const nameRegex = new RegExp('^[a-zA-Z0-9_\\-\\.]*$');
+    const newNameCandidate = event.target.value.replace(/[/\\?%*:|"<>]/g, '-').trim();
+    if (!newNameCandidate) {
+      setErrorMessage(t('left_menu.pages_error_empty'));
+    } else if (newNameCandidate.length >= 30) {
+      setErrorMessage(t('left_menu.pages_error_length'));
+    } else if (!newNameCandidate.match(nameRegex)) {
+      setErrorMessage(t('left_menu.pages_error_format'));
+    } else {
+      setErrorMessage('');
+      setLayoutSetName(newNameCandidate);
+    }
+  };
+
+  return (
+    <div
+      ref={configPanelRef}
+      className={classes.configureLayoutSet}>
+      {editLayoutSetName ? (
+        <>
+          <span>{t('left_menu.configure_layout_sets_name')}</span>
+          <TextField
+          onKeyDown={handleKeyPress}
+          onChange={handleOnChange}
+          defaultValue={layoutSetName}
+          isValid={!errorMessage}
+        />
+        </>) : (
+      <Button
+        className={classes.configureLayoutSetButton}
+        variant={ButtonVariant.Quiet}
+        onClick={handleConfigureLayoutSetButtonClick}>
+        {t('left_menu.configure_layout_sets')}
+      </Button>)
+      }
+      <div
+        className={classes.componentHelpIcon}
+        aria-haspopup="true"
+        onMouseEnter={handlePopoverOpen}>
+        <Button
+          icon={<InformationIcon/>}
+          variant={ButtonVariant.Quiet}
+        />
+      </div>
+      {popoverOpen && (
+        <Popover
+          open={!!popoverOpen}
```

Comment:
Tror ikke `popoverOpen` er nødvedig nødvendig her.
```suggestion
          open={!!anchorEl}
```